src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const noDiscount = days * 40;
```

Comment:
Good job! 🔥 
But it is better to rename noDiscount, it is not clear what it means now. 
I would make it fullPrice